src/scss/custom/_tables.scss

```diff
@@ -14,3 +14,107 @@ th,
 td {
   min-width: 150px;
 }
+
+.model-details__apps {
```

Comment:
I think that all of these changes should probably be in https://github.com/canonical-web-and-design/jaas-dashboard/blob/master/src/pages/Models/Details/_model-details.scss instead. This move should also resolve the stylelint issue.